### DISCUSSION TITLE
How to enable Sign in with Email?

### DISCUSSION THREAD

**Discussion by ersel** [outline/outline#1842] [Self Hosting] [answered]
I have the **users can sign-in using their email address** option enabled under security settings. However, I don't see the sign in with email option on the landing page for my installation?

Only option I see is "Continue with Google". How can I allow access to select users by their email addresses? These users will be using different email providers...

**Comment by ersel** [comment, marked as answer]
Looks like you need to have only one team on the instance for it to work correctly 

https://github.com/outline/outline/issues/1203#issuecomment-669274819

**Reply by ersel** [reply]
Ok, after cleaning up the teams table it's working as expected.

I have one last question. Are magic sign-in links only sent to "invited" users?

**Reply by tommoor** [reply]
That's right, allowing other users this option would potentially bypass 2FA or other authentication controls appliied to GSuite
